src/model/Model.cpp

```diff
@@ -1,12 +1,11 @@
 #include "Model.h"
 #include <iostream>
 
-Model::Model(string name) {
-    tableName = name;
-}
+Model::Model(string name) :
+    tableName(name) {}
 
 Model::~Model() {
-    for (vector<Field*>::const_iterator it = fields.begin(); it != fields.end(); it++) {
+    for (vector<Field*>::const_iterator it = fields.begin(); it != fields.end(); ++it) {
```

Comment:
From what I can tell, this doesn't apply to for-loops declaration, unless you did something like:
`for (int i=0, a=0; i < 10; a = i++) {` vs `for (int j=0, a=0; j < 10; a = ++j) {`
In which case `a` would be affected, but `i`/`j` are equal in both (I may be wrong though).

Otherwise, do we just leave this as it was and ignore the code styling issue (Prefer prefix ++/-- operators for non-primitive types.) or is there another way around this?